src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BooksList from '../containers/BooksList';
+import BooksForm from '../containers/BooksForm';
+
+function App() {
+  return (
+    <div>
+      <BooksList key={12345} />
```

Comment:
- [ ] BookList doesn't end up using the key prop and besides its a container so kindly remove it.